src/main/java/duke/utils/Ui.java

```diff
@@ -0,0 +1,89 @@
+package duke.utils;
+
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Handles the printing of output to the console
+ */
+public class Ui {
+
+
+    public Ui() {
```

Comment:
It seems you may have missed out on writing JavaDoc for this default constructor!